méli-mélo/th-canadaday/meta.md

```diff
@@ -10,10 +12,15 @@ cssClass:
 - brdr-canadaday
 a11yStatement: >
   This colour meets the colour contrast requirements as outlined in WCAG 2.0 AA Success Criterion 1.4.3: Contrast (Minimum).
-  Tested by Nick Frenette, <a href="mailto:nick.frenette@canada.ca">nick.frenette@canada.ca</a>. 2021-06-01.
+  Tested by Nick Frenette, nick.frenette at canada.ca. 2021-06-01.
 peNote:
 - The <code>bg-canadaday</code> class must be accompagnied with another dark contrast background colour such as <code>bg-dark</code>
 - The <code>panel-canadaday</code> class must be accompagnied with a fall back color such as <code>panel-default</code>
-lead: PCH - Nick Frenette (@nfrenette)
+pages:
+  examples:
+    - title: Canada day 2021
+      language: en
+      path: index.html
```

Comment:
This should be `canada-day.html` instead, because `index.html` doesn't exist for this one.

Reply:
fixed